src/main/webapp/assets/modules/employer-dashboard/employerDashboardController.js

```diff
@@ -1,20 +1,55 @@
-techlooper.controller('employerDashboardController', function ($scope, jsonValue, utils, apiService, $location, $filter) {
-  $('.summernote').summernote();
-  var edit = function() {
-    $('.click2edit').summernote({
-      focus: true
-    });
-  };
-  var save = function() {
-    var aHTML = $('.click2edit').code(); //save HTML If you need(aHTML: array).
-    $('.click2edit').destroy();
-  };
+techlooper.controller('employerDashboardController', function ($scope, jsonValue, utils, apiService, $location, $filter, $route) {
```

Comment:
Line is too long.<br>'techlooper' is not defined.<br>'$route' is defined but never used.